packages/dev/core/src/Meshes/Builders/linesBuilder.ts

```diff
@@ -7,6 +7,7 @@ import type { FloatArray, Nullable } from "../../types";
 import { LinesMesh } from "../../Meshes/linesMesh";
 import type { Scene } from "../../scene";
 import { VertexBuffer } from "../../Buffers/buffer";
+import { Tools } from "../../Misc/tools";

```

Comment:
Can we use logger instead of tools as an import ? to keep tree shaking happy :-)

Reply:
Done!